### DISCUSSION TITLE
Multiple songs in one file

### DISCUSSION THREAD

**Discussion by janproch** [SongProOrg/songpro.org#41] [General]
Hello,

it is possible to have more songs in one file? Chordpro has new_song directive, if I understand correctly

I am considering songpro as primary format for songbook app https://songi.app/ which I created. I like songpro simplicity, good documentation, it seems as good format to be directly modified by user. I want to have all songs from one songbook in one big text file. I could use eg. --- on separate line as song separator, but if there will be official support in songpro format, it could have benefits like easy song database portability between different software.

Thanks

**Comment by spilth** [comment]
Hi Jan,

I don't currently have any plans to add multi-song support but it's something I'll consider.

I agree that I would likely use `---` as a song delimiter and probably make a separate parser that passes each song off to the single-song parser.
